hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -193,9 +193,8 @@ private void lookIfUpdateAvailable(final Target target) {
         }
 
         if (action.get().isCancelingOrCanceled()) {
-            amqpMessageDispatcherService.targetCancelAssignmentToDistributionSet(
-                    new CancelTargetAssignmentEvent(target.getOptLockRevision(), target.getTenant(),
-                            target.getControllerId(), action.get().getId(), target.getTargetInfo().getAddress()));
+            amqpMessageDispatcherService.targetCancelAssignmentToDistributionSet(new CancelTargetAssignmentEvent(
+                    target.getOptLockRevision(), target.getTenant(), target, action.get().getId()));
```

Comment:
Why does that method signature of CancelTargetAssignmentEvent now has the parameter OptLockRevision, tenant and target. Target has already all the information. I would prefer either to only have the target and actionId in the method signature or the only the necessary parameter of target.